### ISSUE TITLE
New Native Swift project fails to compile

### ISSUE THREAD

**Issue by colemcampbell** [forcedotcom/SalesforceMobileSDK-iOS#3342]
Please fill out the following details:

1. Version of Mobile SDK Used: 9.0 (Also happens with 8.3)
2. Issue found in Native App or Hybrid App: Native App (iOS)
3. Xcode Version: Version 12.4 (12D4e)
4. OS Version: MacOS 11.2.1 (20D74)
5. Device: MacBook Pro (16-inch, 2019)
6. Steps to reproduce:
- Create project with `forceios create`
- Choose `native_swift`
- Complete installation, open the created workspace
- Build project
7. Expected Behavior: successful build
8. Actual behavior:
- Build fails with 8 undefined symbol errors

<img width="550" alt="Screen Shot 2021-02-26 at 1 38 31 PM" src="https://user-images.githubusercontent.com/12163026/109346944-04dc9a00-7838-11eb-8c3c-9bc007189595.png">

9. Error Log:
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_SFSDKSafeMutableSet", referenced from:
      objc-class-ref in SFRestAPI.o
  "_OBJC_CLASS_$_SFJsonUtils", referenced from:
      objc-class-ref in SFIdentityCoordinator.o
      objc-class-ref in SFOAuthCoordinator.o
      objc-class-ref in SFRestAPI+Files.o
      objc-class-ref in SFRestAPI.o
      objc-class-ref in SFRestRequest.o
      objc-class-ref in SFSDKAuthConfigUtil.o
      objc-class-ref in SFSDKOAuth2.o
      ...
  "_OBJC_CLASS_$_SFLogger", referenced from:
      _OBJC_CLASS_$_SFSDKCoreLogger in SFSDKCoreLogger.o
  "_OBJC_CLASS_$_SFSDKDatasharingHelper", referenced from:
      objc-class-ref in SFDirectoryManager.o
      objc-class-ref in SFUserAccountManager.o
  "_OBJC_METACLASS_$_SFLogger", referenced from:
      _OBJC_METACLASS_$_SFSDKCoreLogger in SFSDKCoreLogger.o
  "_OBJC_CLASS_$_SFFileProtectionHelper", referenced from:
      objc-class-ref in SFDefaultUserAccountPersister.o
      objc-class-ref in SFDirectoryManager.o
  "_OBJC_CLASS_$_SFSDKSafeMutableDictionary", referenced from:
      objc-class-ref in SalesforceSDKManager.o
      objc-class-ref in SFNetwork.o
      objc-class-ref in SFRestAPI.o
      objc-class-ref in SFUserAccountManager.o
  "_OBJC_CLASS_$_SFSwiftDetectUtil", referenced from:
      objc-class-ref in SalesforceSDKManager.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by wmathurin** [comment]
Hmm, can't repro.
I just ran `forceios create`, created a native swift app, opened the workspace in Xcode and build (and ran) the app without issues.
My versions:
- forceios 9.0.0
- cocoapods 1.10.1
- xcode 12.4
- macOS 10.15.7 (catalina)

**Comment by wmathurin** [comment]
The only notable difference between your setup and mine is the macOS version. Let me see if we have a Big Sur machine to try it on.

**Comment by wmathurin** [comment]
This [comment](https://stackoverflow.com/questions/63607158/xcode-12-building-for-ios-simulator-but-linking-in-object-file-built-for-ios/64139830#64139830) has an explanation of what is going on as well as a fix.

**Comment by wmathurin** [comment]
We will do the fix in the templates for 9.1, but in the meantime, we can do the fix recommended:
> a simple workaround to this issue is to set "Build Active Architecture Only (ONLY_ACTIVE_ARCH)" to Yes in your libraries/apps, even for release mode.

**Comment by wmathurin** [comment]
Fixed with https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/314.
Will be in our next release.
